rafter/step1.md

```diff
@@ -1,20 +1,7 @@
+The environment is set up and ready to start playing with Rafter.
 
-Make sure the environment is set up. Check if the API Server is running. 
-It make take at least 1 minute, Minikube needs time, be patient.
-`kubectl get pod -n kube-system | grep kube-apiserver-minikube`{{execute}}
+Install Rafter from the dedicated Helm repository. Run:
 
-Initialize Helm to get Tiller running in Minikube
-`kubectl -n kube-system create serviceaccount tiller`{{execute}}
-`kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller`{{execute}}
-`helm init --service-account=tiller`{{execute}}
-
-Check if Tiller is running
-`kubectl get deploy -n kube-system | grep tiller-deploy`{{execute}}
-
-Install Rafter from dedicated Helm repository
 `helm repo add rafter-charts https://rafter-charts.storage.googleapis.com`{{execute}}
```

Comment:
I suggest to make 2 steps, one `Add new Chart's repository to Helm` and the `Install Rafter`